freqtrade/optimize/backtesting.py

```diff
@@ -254,7 +257,8 @@ def _get_ohlcv_as_lists(self, processed: Dict[str, DataFrame]) -> Dict[str, Tupl
         self.progress.init_step(BacktestState.CONVERT, len(processed))
 
         # Create dict with data
-        for pair, pair_data in processed.items():
+        for pair in list(processed.keys()):
```

Comment:
what's the reason to explicitly cast to `list` here?
i think you can iterate `keys()` (which will return object with dict_keys) just fine without explicit cast?

Reply:
Oh yeah this is a leftover from when i used `del`. Had to make a copy of keys so `processed` could be modified during iteration, and forgot to remove it when i removed `del`. More on that in the next response.